js_modules/dagster-ui/packages/ui-core/src/assets/AssetDefinedInMultipleReposNotice.tsx

```diff
@@ -24,12 +25,15 @@ export const AssetDefinedInMultipleReposNotice = ({
   loadedFromRepo: RepoAddress;
   padded?: boolean;
 }) => {
-  const {data} = useQuery<AssetDefinitionCollisionQuery, AssetDefinitionCollisionQueryVariables>(
-    ASSET_DEFINITION_COLLISION_QUERY,
-    {
-      variables: {assetKeys: [{path: assetKey.path}]},
-    },
-  );
+  const queryResult = useQuery<
+    AssetDefinitionCollisionQuery,
+    AssetDefinitionCollisionQueryVariables
+  >(ASSET_DEFINITION_COLLISION_QUERY, {
+    variables: {assetKeys: [{path: assetKey.path}]},
+  });
+  const {data} = queryResult;
+
+  useBlockTraceOnQueryResult(queryResult, 'AssetDefinitionCollisionQuery');
```

Comment:
I'm not sure about this one -- on some pages there are small components that use separate queries specifically because they're slow and don't really block usability of the page.

eg: I think this one may be slow and I don't think that means "the asset view takes a long time to load and we should prioritize fixing it" -- not sure what we should do in that case... maybe we just ignore these later in DD?

Reply:
Yeah I wasn't sure tbh, I'll remove this one for now.